src/grpc/lib-oc/OpenConfigNpuUtilization.cpp

```diff
@@ -41,25 +41,19 @@ OpenConfigNpuUtilization::iterate (JuniperNetworksSensors *handle, Telemetry::Op
 
             kv = datap->add_kv();
             kv->set_key("memory/" + mem.name() + "/lowest_util");
-            kv->set_int_value(mem.highest_util());
+            kv->set_int_value(mem.lowest_util());
 
-            if (mem.average_cache_hit_rate()) {
-                kv = datap->add_kv();
-                kv->set_key("memory/" + mem.name() + "/average_cache_hit_rate");
-                kv->set_int_value(mem.average_cache_hit_rate());
-            }
+            kv = datap->add_kv();
```

Comment:
During our discussions we felt memory cache hit rates being zero should also be exported as this maybe a reason for concern on platforms where line rate cannot be met unless a certain cache hit rate is achieved.

Across the sensors both in case of UDP export or GRPC we discussed with testing teams and decided not skip selective fields within a record if they are zero.
i.e. say lowest cache hit rate is 0, 
highest cache hit rate is 70
Avg cache hit rate is 40

We export all three fields 0, 70 and 40 respectively instead of skipping just lowest cache hit rate of 0. Because just exporting 70, 40 becomes confusing for the user.